pubspec.yaml

```diff
@@ -1,17 +1,17 @@
 name: git_hooks
 description: Using dart to prevents bad commit or push (git hooks, pre-commit/precommit, pre-push/prepush, post-merge/postmerge and all that stuff...).It is similar to husky.
-version: 0.1.4
+version: 0.2.0
```

Comment:
Null-safe versions are breaking changes so the `0.2.0` version change is justified here.